symfony/framework-bundle/3.3/manifest.json

```diff
@@ -15,7 +15,7 @@
         "APP_ENV": "dev",
         "APP_SECRET": "%generate(secret)%",
         "#TRUSTED_PROXIES": "127.0.0.1,127.0.0.2",
-        "#TRUSTED_HOSTS": "localhost,example.com"
+        "#TRUSTED_HOSTS": "'^localhost|example\\.com$'"
```

Comment:
Isn't it missing braces i.e. `"'^(localhost|example\\.com)$'"` (or alternatively `"'^localhost$|^example\\.com$'"`)?

Reply:
please try locally and send a PR if this doesn't work as expected

Reply:
As is it would match `localhost` and `example.com` but also e.g. `localhostfoo` and `fooexample.com`. Proposed #728 as a fix